meta-lmp-bsp/recipes-kernel/kernel-modules/kernel-module-nxp89xx_git.bbappend

```diff
@@ -1 +1 @@
-INHIBIT_PACKAGE_DEBUG_SPLIT = "${@bb.utils.contains('DISTRO_FEATURES', 'modsign', '1', '', d)}"
+inherit ${@bb.utils.contains('DISTRO_FEATURES', 'modsign', 'kernel-modsign', '', d)}
```

Comment:
I think INHIBIT_PACKAGE_DEBUG_SPLIT was not yet backported to kirkstone (in meta-security), right?

Reply:
right, I will drop the last patch and add it later when backported to kirkstone

Reply:
One other minor thing, can you move this append under meta-lmp-bsp/dynamic-layers/freescale-layer? Otherwise without that layer we will get a warning.

Reply:
fixed and thanks for pointing about that